test/cypress/package.json

```diff
@@ -36,9 +36,13 @@
     "eslint": "^8.17.0",
     "eslint-plugin-cypress": "^2.12.1",
     "execa": "^6.1.0",
+    "lint-staged": "^13.0.3",
     "prettier": "^2.7.1",
     "rimraf": "^3.0.2",
     "start-server-and-test": "^1.14.0"
   },
-  "packageManager": "yarn@3.2.1"
+  "packageManager": "yarn@3.2.1",
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": "../../.husky/pre-commit-check-for-skipped-tests"
```

Comment:
I didn't install prettier and eslint this time (just lint-staged to get a tool that knows which files are staged for commit), but yep, I agree. Maybe we should pull some of this to the top level, even. It might be a bigger task, though, and maybe something we should do when/if cleaning up the deep folder structure left over from splitting the repos.

Reply:
Theres several structural things we should improve, f.ex remove the /src/shared directory. Ideally perhaps get everything moved up one level, with the end result of 1 package.json?

Anyway, lets leave it as-is for now 🙂 Most important is to remove the .only and the git hook is a great addition!